packages/discord.js/src/client/actions/PresenceUpdate.js

```diff
@@ -6,18 +6,18 @@ const { Events } = require('../../util/Constants');
 class PresenceUpdateAction extends Action {
   handle(data) {
     let user = this.client.users.cache.get(data.user.id);
-    if (!user && data.user?.username) user = this.client.users._add(data.user);
+    if (!user && data.user.username) user = this.client.users._add(data.user);
     if (!user) return;
 
-    if (data.user?.username) {
+    if (data.user.username) {
```

Comment:
#7248 fixed it. So ig you can leave it as it is.

Reply:
Rebased pr to fix it :)